themes/root/theme.config.php

```diff
@@ -22,7 +22,6 @@
             'VuFind\View\Helper\Root\ContentBlock' => 'Laminas\ServiceManager\Factory\InvokableFactory',
             'VuFind\View\Helper\Root\Context' => 'Laminas\ServiceManager\Factory\InvokableFactory',
             'VuFind\View\Helper\Root\Csp' => 'VuFind\View\Helper\Root\CspFactory',
-            'VuFind\View\Helper\Root\CspNonce' => 'VuFind\View\Helper\Root\CspNonceFactory',
```

Comment:
We should probably leave this here for now, since the helper is deprecated but not removed. I'll add a TODO reminder that we should remove the helper, this line, and also the alias line below in VuFind 11.

Reply:
fixed